.changeset/mighty-coats-breathe.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kuma-ui/sheet": minor
```

Comment:
@myzkyy I think the patch version is better because this is a bugfix.

Reply:
@kotarella1110 Fixed 477c3ffa64ea19d7ad63e94e2eeb8bad12e18b82